### DISCUSSION TITLE
Wat ga je doen vandaag? - 18 juni

### DISCUSSION THREAD

**Discussion by D0nnaz** [Sensinki/Blok-Tech-Team-4A#19] [Wat ga je doen vandaag?]
Ik ga verder aan het minifyen en uglifyen. Het uglifyen is als gelukt en het minifyen deels zie https://github.com/Sensinki/Blok-Tech-Team-4A/issues/9. Misschien weten jullie waarom dit gebeurt? Ik ga het straks in het tech support kanaal gooien.

**Comment by D0nnaz** [comment]
Ik weet al waarom het gebeurt, jullie hoeven er niet meer naar te kijken :)

**Reply by dinhs-frontend** [reply]
Topie!

**Comment by dinhs-frontend** [comment]
Ik ga de code in de main weer even bekijken, voor eventuele vragen. Met Sena bespreken wat voor haar nog kan doen. En als er nog iets is wat ik kan doen, stuur maar gerust een berichtje. En anders ga ik nog even aan mijn individuele opdracht werken.

**Reply by D0nnaz** [reply]
Ik dacht dat het misschien slim is om de taken met Sena gaat verdelen. Dus 1 iemand zorgt dat de juiste chats in beeld komen en de andere zorgt dat je maar 1x, per account geredirect wordt naar de /match. Dan moeten jullie zelf maar even kijken wie wat doet. Als dat helemaal klaar is kunnen jullie alvast een begin maken met alles in het MVC model te zetten.

**Reply by Sensinki** [reply]
Ja, eigenlijk was ik ook plan van verdelen van taken. Als je zorgt (@dinhs-frontend) elke gebruiker alleen maar 1x geredirect wordt naar de /match, denk ik dat kunnen we vandaag of uiterlijk maandag die deel klaar hebben

**Comment by Sensinki** [comment]
Vandaag zal ik, afhankelijk van de games die gebruiker leuk vindt, ervoor zorgen dat de chats van die games op de "/" page verschijnen. Als ik hier niet uit kan, doe ik het morgen en vandaag ga ik beginnen met MVC model.

**Reply by D0nnaz** [reply]
Top! Zou nog even overleggen met Sang dat hij het andere deel gaat doen :)

Zag net je reply (my bad)

**Reply by Sensinki** [reply]
Ja, we hebben al besproken 😎

**Reply by D0nnaz** [reply]
![thumbs-up-double-thumbs-up](https://github.com/Sensinki/Blok-Tech-Team-4A/assets/92280247/bc17a025-b4c7-4a44-80ac-88bd9a9c3693)

**Comment by D0nnaz** [comment]
@Sensinki @dinhs-frontend Is het gelukt?

**Reply by Sensinki** [reply]
Nee, ik ben er nog mee bezig, ik wil het voor morgen af hebben. anders ja, nu kan ik ook naar het mvc-model kijken :(
